mockredis/client.py

```diff
@@ -301,11 +302,32 @@ def _rename(self, old_key, new_key, nx=False):
     def dbsize(self):
         return len(self.redis.keys())
 
-    # String Functions #
+    def _decode(self, value):
```

Comment:
All the decode=True's lead to this decode function, that doesn't necessarily decode in the end, but only decodes if decode responses is true.. so I wonder if the "decode" arguments and "_decode" are named well.. but I don't have a better name so idk..

Reply:
that's similar to the _encode function which(which does nothing on string), and thats the action that's done according to client request.

Reply:
Don't like that methods named decode sometimes return stuff that aren't decoded, but ok